packages/next-auth/src/providers/strava.ts

```diff
@@ -0,0 +1,60 @@
+import type { OAuthConfig, OAuthUserConfig } from "."
+
+export interface StravaProfile extends Record<string, any> {
+  id: string // this is really a number
+  username: string
+  resource_state: number
+  firstname: string
+  lastname: string
+  bio: string
+  city: string
+  state?: any
+  country?: any
+  sex: string
+  premium: boolean
+  summit: boolean
+  created_at: string
+  updated_at: string
+  badge_type_id: number
+  weight: number
+  profile_medium: string
+  profile: string
+  friend?: any
+  follower?: any
```

Comment:
Added these in case anyone needs these types for debugging the response. Not sure if adding these to the `profile` return is a convention.

Reply:
The only important fields to document are the ones read in the profile callback below:
```ts
profile(profile) {
      return {
        id: profile.id,
        name: `${profile.firstname} ${profile.lastname}`,
        email: null,
        image: profile.profile,
      }
    },
```
hence let's remove the others for readability.